README.md

```diff
@@ -124,6 +123,15 @@ clients:
       - https://api-2c.example.com/api
 ```
 
+## Retry Behavior
+Conjure-go-runtime HTTP clients provide the following retry behavior:
+- HTTP Status Code Handling
+  - 429 (Too Many Requests): the client retries the request to a different node based on its URI configuration. TODO: support deriving backoff from the retry-after HTTP response header
+  - 503 (Service Unavailable): the client retries the request to a different node based on its URI configuration.
+  - 307 (Temporary Redirect), 308 (Permanent Redirect): the client retries the request with the URL included in the HTTP response's 'Location' header.
```

Comment:
nit: Mind placing this above the 429 so we have codes in ascending order. I just find it easier to read when it's in order.